### ISSUE TITLE
Problem to convert list to Pandas from Raw

### ISSUE THREAD

**Issue by pymishi** [cognitedata/cognite-sdk-python#547]
**Describe the bug**
Converting a list to pandas from RAW List is raising an error. This looks to be related to Null values in columns.

**To Reproduce**
Runnable code reproducing the error.
```
from cognite.client import CogniteClient
from cognite.client.data_classes import Asset

c = CogniteClient(api_key=dbutils.secrets.get('dlc-test', 'databricks-scripts'), client_name='w-dea_assets-pipeline', project='w-dea-test')

raw_assets = c.raw.rows.list(db_name='LandingZone', table_name='IH08', limit=-1)

df = raw_assets.to_pandas()
```
**Expected behavior**
Raw List should be converted to Pandas

**Screenshots**
![image](https://user-images.githubusercontent.com/47743978/65313097-1aa25600-db94-11e9-9422-0778ac746b5c.png)

**System information (please complete the following information):**
 - OS: Windows and DataBricks
 - Python Version: n/a
 - SDK Version: 1.1.8

**Additional context**

**Comment by pymishi** [comment]
![image](https://user-images.githubusercontent.com/47743978/65313303-6fde6780-db94-11e9-9372-611c9ef51f85.png)
The image above shows the column in red.

![image](https://user-images.githubusercontent.com/47743978/65313409-97353480-db94-11e9-899d-748414d94eae.png)
The image above shows that the previous column does not exist.

**Comment by haakonvt** [comment]
Has anyone started this? This is a high priority for a customer and I'd like to be assigned to this unless a fix is already in the works 😊

**Comment by haakonvt** [comment]
Fix released with `v1.3.1` release #570
